ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -130,18 +130,20 @@ const isReformatInvalid = (deviceData, request, requests) => {
     }
 };
 
-const isDeviceMountPointInvalid = (deviceData, request) => {
+const isDeviceMountPointInvalid = (deviceData, requiredMountPoints, request) => {
     const device = request["device-spec"];
+    const requiredMountPointData = requiredMountPoints.find(val => val.value.v === request["mount-point"]);
 
     if (!device || !request["mount-point"]) {
         return [false, ""];
     }
```

Comment:
I'm not 100% sure, but I think you also want to guard against no matching required mount point? It seems to me you'd get a "null dereference" in the code below (whatever it's called in javascript).